### ISSUE TITLE
/node/add - ul elements need a set of li children

### ISSUE THREAD

**Issue by martinfrances107** [jsdrupal/drupal-admin-ui#342]
Screenreaders are very sensitive to the DOM structure of ol, ul elements 

Failure to maintain  a ul li parent-child relationship breaks accessibility.

At the moment we only have a set of A elements as the direct children of a UL element 

The images below shows that the axe report for the /node/add page.

It needs a little TLC.

![axeulissue](https://user-images.githubusercontent.com/1259969/44206825-fa16f980-a152-11e8-8054-dd0de415a5a9.png)
.

I have fixed a similar issue so I know what the problem is. It should be a one line fix.. I am working on this now.

**Comment by martinfrances107** [comment]
Thank you, for the rapid response as always. 💯
